hypervisor/include/arch/x86/vm_config.h

```diff
@@ -17,7 +17,7 @@
 
 #define CONFIG_MAX_VM_NUM	(PRE_VM_NUM + SOS_VM_NUM + MAX_POST_VM_NUM)
 
-#define AFFINITY_CPU(n)		(1U << (n))
+#define PCPU_BITMAP(n)		(1UL << (n))
```

Comment:
Using MACRO of AFFINITY_CPU is still fine to me.  Not?

Reply:
I was thinking the pcpu_affinity_bitmap may represent the actual CPU affinity definition, or a superset of pCPU for post-launched. So probably PCPU_BITMAP maybe a more neutral term?

Reply:
ok